src/pages/mint/index.js

```diff
@@ -119,34 +119,42 @@ export const Mint = () => {
               <Input
                 type="text"
                 onChange={(e) => setTitle(e.target.value)}
-                placeholder="title"
```

Comment:
Is there a reason why you're changing the <Input /> API and removing the placeholder text? with API changes like this I would like to get a rational on why we need to change it?

Reply:
Good question. "Placeholder" has a specific meaning in HTML inputs, which is what it was used for before. You can still use placeholder how it was used earlier. I just added `label` which adds a `<Label/>` around the input.